app/javascript/dashboard/helper/snoozeHelpers.js

```diff
@@ -1,11 +1,36 @@
-import { getUnixTime, format } from 'date-fns';
-import * as chrono from 'chrono-node';
+import { getUnixTime, format, add } from 'date-fns';
+import wootConstants from 'dashboard/constants/globals';
+
+const SNOOZE_OPTIONS = wootConstants.SNOOZE_OPTIONS;
 
 export const findSnoozeTime = snoozeType => {
-  const parsedDate = chrono.parseDate(snoozeType);
+  let parsedDate = null;
+
+  const currentDate = new Date();
+  const addTime = time => {
+    return new Date(
+      currentDate.getFullYear(),
+      currentDate.getMonth(),
+      currentDate.getDate() + time,
+      9,
+      0,
+      0,
+      0
+    );
+  };
```

Comment:
We can use `add` method from `date-fns`  for this instead of this implementation